README.md

```diff
@@ -26,6 +26,10 @@
 <a href="https://saucelabs.com/u/sinonjs"><img src="https://saucelabs.com/browser-matrix/sinonjs.svg" alt="Sauce Test Status"></a>
 </p>
 
+## Compatibility
+
+For details on compatibility, please see [`COMPATIBILITY.md`][COMPATIBILITY.md]
```

Comment:
```suggestion
For details on compatibility and browser support, please see [`COMPATIBILITY.md`][COMPATIBILITY.md]
```

Reply:
I am not sure people looking for which browsers are supported, would associate this with compatibility when casually glancing or searching for hints. Might as well be specific.

Reply:
True story